region/maridia/inner-pink.json

```diff
@@ -585,14 +585,9 @@
             },
             {
               "id": 6,
-              "requires": [
-                {"or":[
-                  "canSwim",
-                  "HiJump",
-                  "canSpringBallJumpMidAir"
-                ]}
-              ],
-              "unlock": null
+              "requires": null,
+              "unlock": null,
+              "note": "A jump assist would be needed here if there were no snails to climb on"
```

Comment:
Shouldn't this be hidden behind some tech?

Reply:
I wasn't sure. It's too simple to be a tech. The only reason to have one is for Enemizer way down the line. I decided it could be done later when needed, but not 100% sure. How do you see it?